code/stable/gamephys/SRS/Chipmunk_SRS.tex

```diff
@@ -885,13 +891,57 @@ \subsubsection{Data Definitions}
               \item{${\mathbf{v}^{BP}}$ is the velocity of the point of collision P in body B ($\frac{\text{m}}{\text{s}}$)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & In a collision, the velocity of a rigid body \hyperref[assumpOT]{A: objectTy} A colliding with another rigid body B relative to that body ${{\mathbf{v}_{i}}^{AB}}$ is the difference between the velocities of A and B at point P.
+Notes & In a collision, the velocity of a rigid body \hyperref[assumpOT]{A: objectTy} A colliding with another rigid body B relative to that body ${{\mathbf{v}_{i}}^{AB}}$ is the difference between the velocity of A and B at point P.
+\\ \midrule \\
+Source & --
+\\ \midrule \\
+RefBy & 
+\\ \bottomrule \end{tabular}
+\end{minipage}
+\par~
+
+\noindent \begin{minipage}{\textwidth}
+\begin{tabular}{p{0.2\textwidth} p{0.73\textwidth}}
+\toprule \textbf{Refname} & \textbf{DD:impulseV}
+\phantomsection 
+\label{DD:impulseV}
+\\ \midrule \\
+Label & Impulse (vector)
+\\ \midrule \\
+Symbol & $\mathbf{J}$
+\\ \midrule \\
+Units & Ns
+\\ \midrule \\
+Equation & \begin{displaymath}
+           \mathbf{J}=m Δ\mathbf{v}
+           \end{displaymath}
+\\ \midrule \\
+Description & \begin{symbDescription}
+              \item{$\mathbf{J}$ is the impulse (vector) (Ns)}
+              \item{$m$ is the mass (kg)}
+              \item{$Δ\mathbf{v}$ is the velocity ($\frac{\text{m}}{\text{s}}$)}
+              \end{symbDescription}
+\\ \midrule \\
+Notes & An impulse (vector) $\mathbf{J}$ occurs when a force $\mathbf{F}$ acts over a body over an interval of time Derivation of impulse (vector).
+        \hyperref[assumpOT]{A: objectTy}
 \\ \midrule \\
 Source & --
 \\ \midrule \\
 RefBy & 
 \\ \bottomrule \end{tabular}
 \end{minipage}
+ Derivation of impulse (vector) - Newton's second law of motion states:
+\begin{displaymath}
+\mathbf{F}=m \mathbf{a}=m \frac{d\,Δ\mathbf{v}}{d\,t}
+\end{displaymath}
+Rearranging :
+\begin{displaymath}
+\int_{{t_{1}}}^{{t_{2}}}{\mathbf{F}}\,dt=m \left(\int_{{\mathbf{v}_{1}}}^{{\mathbf{v}_{2}}}{1}\,d\mathbf{v}\right)
```

Comment:
the latex is fine - maybe it was the html you were showing?

Reply:
The latex is fine, its just the html that looks kind of weird